pages/resources/guild.mdx

```diff
@@ -679,13 +679,16 @@ Additional information about a participating user's join source in a [guild](/re
 
 ###### Supplemental Guild Member Structure
 
-| Field              | Type       | Description                                                             |
-| ------------------ | ---------- | ----------------------------------------------------------------------- |
-| user_id            | snowflake  | The ID of the user this guild member represents                         |
-| join_source_type   | integer    | [How the user joined the guild](#join-source-type)                      |
-| source_invite_code | ?string    | The invite code or vanity used to join the guild, if applicable         |
-| inviter_id         | ?snowflake | The ID of the user who invited the user to the guild, if applicable     |
-| integration_type   | ?integer   | The type of integration that added the user to the guild, if applicable |
+| Field              | Type                                        | Description                                                             |
+| ------------------ | ------------------------------------------- | ----------------------------------------------------------------------- |
+| user_id? ^1^       | snowflake                                   | The ID of the user this guild member represents                         |
+| member? ^1^        | [guild member](#guild-member-object) object | The associated guild member                                             |
+| join_source_type   | integer                                     | [How the user joined the guild](#join-source-type)                      |
+| source_invite_code | ?string                                     | The invite code or vanity used to join the guild, if applicable         |
+| inviter_id         | ?snowflake                                  | The ID of the user who invited the user to the guild, if applicable     |
+| integration_type   | ?integer                                    | The type of integration that added the user to the guild, if applicable |
+
+^1^ `user_id` is only included when fetched from the [Get Guild Members Supplemental](#get-guild-members-supplemental) endpoint. `member` is only included when fetched from the [Search Guild Members](#search-guild-members) endpoint.
```

Comment:
Why not have different footnotes for these

Reply:
done